.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: simple
+          extra-files: |
+            ./VERSION
```

Comment:
Need to confirm that this actually bumps the version contained in this file before the 👇🏻  build step happens. 

see: https://github.com/googleapis/release-please/blob/main/docs/customizing.md#updating-arbitrary-files

I think it will.

Reply:
When you say bump the version, if it ups the version number it will trigger the update prompt in Lamp Clients but wont update the actual comparison. 

Line 89 of lampFS.h is the client's version number.  Using the release please lamp's version checking will need to be updated.

Ill have a look into doing it.

Reply:
if you mean here https://github.com/CHollingworth/Lamp/blob/6c529ea767ba03bbff7aa42215446222e491417e/Lamp/Filesystem/lampFS.h#L335

then release please can modify that line for you via annotation comments that guide it on what line or lines to change. see the [link about modifying files](https://github.com/googleapis/release-please/blob/main/docs/customizing.md#updating-arbitrary-files).

I still testing the final step: "merge the release PR and upload a built binary" here: https://github.com/airtonix/Lamp/actions/runs/6513666326/job/17693666331

But i can already see one issue that needs more work: 

- your initial version prior to my PR was `1.0.8` 
- release please decided that it would just make it `1.0.0`

I'll look into ways that we can coerce it into using the value already there as a starting point. Perhaps it's looking for a git release tag of the same value? `/unsure`

Reply:
solved that here https://github.com/airtonix/Lamp/blob/feat/46-provide-automated-build-system/Lamp/Filesystem/lampFS.h#L335